packages/monorepo-tools/utils/find-monorepo-root.js

```diff
@@ -0,0 +1,14 @@
+const findUp = require('find-up');
+const fs = require('fs');
+const path = require('path');
+
+async function findMonorepoRoot() {
+  const packagePath = await findUp('package.json');
```

Comment:
we can't find any `package.json` it would fail always when called from a script in a package.json of  a workspace package, which is a use case for the scripts.

find-up should let us pass a fn to check if the we like its content, or you can look for `package-lock.json` instead, basically saying  that is safer to assume it is only in the monorepo root